agent/src/agent/pipeline/config/jython_scripts/observium.py

```diff
@@ -0,0 +1,157 @@
+global sdc
+
+try:
+    sdc.importLock()
+    import sys
+    import os
+    import time
+    import traceback
+
+    sys.path.append(os.path.join(os.environ['SDC_DIST'], 'python-libs'))
+    import requests
+
+    from datetime import datetime, timedelta
+    from requests.auth import HTTPBasicAuth
+finally:
+    sdc.importUnlock()
+
+# single threaded - no entityName because we need only one offset
+entityName = ''
+LAST_TIMESTAMP = '%last_timestamp%'
+
+
+POLL_TIME_KEYS = {
+    'ports': 'poll_time',
+    'mempools': 'mempool_polled',
+    'processors': 'processor_polled',
+    'storage': 'storage_polled'
+}
+
+DATA_KEYS = {
+    'ports': 'ports',
+    'mempools': 'entries',
+    'processors': 'entries',
+    'storage': 'storage'
+}
+
+
+def get_now_with_delay():
+    return int(time.time()) - int(sdc.userParams['DELAY_IN_SECONDS'])
+
+
+def to_timestamp(date):
+    epoch = datetime(1970, 1, 1)
+    return int((date - epoch).total_seconds())
+
+
+def get_interval():
+    return int(sdc.userParams['INTERVAL_IN_SECONDS'])
+
+
+def _get(url, params, response_key):
+    for i in range(1, N_REQUESTS_TRIES + 1):
+        try:
+            res = requests.get(
+                url,
+                auth=HTTPBasicAuth(sdc.userParams['API_USER'], sdc.userParams['API_PASSWORD']),
+                params=params,
+                verify=bool(sdc.userParams['VERIFY_SSL']),
+                timeout=sdc.userParams['QUERY_TIMEOUT'],
+            )
+            res.raise_for_status()
+            return res.json()[response_key]
+        except requests.HTTPError as e:
+            requests.post(sdc.userParams['MONITORING_URL'] + str(res.status_code))
+            sdc.log.error(str(e))
+            if i == N_REQUESTS_TRIES:
+                raise
+            time.sleep(2 ** i)
+
+
+def get_data():
```

Comment:
this function is used one time and has a very generic name, are you sure you need it?